book/book.toml

```
@@ -3,10 +3,10 @@ authors = ["Blaž Hrastnik"]
 language = "en"
 multilingual = false
 src = "src"
-edit-url-template = "https://github.com/helix-editor/helix/tree/master/book/{path}?mode=edit"
 
 [output.html]
 cname = "docs.helix-editor.com"
 default-theme = "colibri"
 preferred-dark-theme = "colibri"
 git-repository-url = "https://github.com/helix-editor/helix"
+edit-url-template = "https://github.com/helix-editor/helix/tree/master/book/{path}?mode=edit"
```

```suggestion
edit-url-template = "https://github.com/helix-editor/helix/tree/master/book/{path}"
```

looks like that part is from the bitbucket example, I don't think it has any effect for github links: https://rust-lang.github.io/mdBook/format/configuration/renderers.html#html-renderer-options

---

[a7eedb9](https://github.com/helix-editor/helix/pull/7278/commits/a7eedb94023eec5690dbcbcff7cc6b9604e9d152)